homeassistant/components/harmony/remote.py

```diff
@@ -178,6 +141,39 @@ def __init__(self, name, host, port, activity, out_path, delay_secs):
         self._delay_secs = delay_secs
         self._available = False
 
+    @property
```

Comment:
Please remove all the properties that just return an instance attribute. Just use regular instance attributes instead.

Reply:
changed.